legal-api/flags.json

```diff
@@ -27,6 +27,15 @@
     },
     "dbc-enabled-business-types": {
       "types": ["SP"]
-    }
+    },
+    "enabled-business-summary-entities": "CP",
+    "supported-amalgamation-out-entities": "BC",
+    "supported-consent-amalgamation-out-entities": "BC",
+    "supported-put-back-on-entities": "BC",
+    "supported-continuation-in-entities": "BC",
+    "support-agm-extension-entities": "BC",
+    "supported-agm-location-change-entities": "BC",
+    "supported-continuation-out-entities": "BC",
+    "supported-consent-continuation-out-entities": "BC"
```

Comment:
These fallback values will be used if LEAR can't get the FFs from LaunchDarkly.

I think most of these should be empty strings, since we haven't fully released these features, and especially not for BC Limited companies.

Reply:
These are not fallback values. It is for local only. Ref: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/flags.py#L71